src/extensions/extensionsHelper.ts

```diff
@@ -405,6 +411,7 @@ export async function publishExtensionVersionFromLocalSource(args: {
   rootDirectory: string;
   nonInteractive: boolean;
   force: boolean;
+  stage: "rc" | "alpha" | "beta" | "stable";
```

Comment:
Can you check what the behavior is when you pass something other than these options? AFAICT, there is nothing in place rn to prevent users from passing other values.

Reply:
Added explicit check.